build.gradle

```diff
@@ -73,16 +74,30 @@ subprojects {
         mavenCentral()
         mavenLocal()
     }
-
-    checkstyle {
-        toolVersion = '8.12'
-        configFile = rootProject.file('config/checkstyle/checkstyle.xml')
+    
+    spotless {
+        java {
+            target project.fileTree(project.rootDir) {
+                include '**/*.java'
+                exclude '**/package-info.java'
+            }
+            importOrder 'cern', 'com', 'io', 'org', 'reactor', 'java', 'javax', 'static *'
+            eclipse().configFile 'config/checkstyle/micrometer-style.xml'
```

Comment:
Should this still be in a "checkstyle" subdir instead of "spotless"? (Just asking...)

Reply:
Originally, I was thinking of `checkstyle` as library-independent name that could prevent folder/file names change if we would go with other library in future. But since there is product that called the same way, it's probably makes it more confusing. Changed to spotless.